CHANGES.rst

```diff
@@ -25,6 +25,9 @@ astropy.extern
 astropy.io.ascii
 ^^^^^^^^^^^^^^^^
 
+- Enable processing of CSV files generated by pandas that have an empty label
+  for the index column. [#6722]
```

Comment:
I'm sorry to mess up your PR number...

Reply:
@bsipocz it's okay, there's apparently a race condition in `next_pr_number.py` 😛